pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
+
+`yacas`
+
+- In the console session, execute a statement:
+
+`In> {{Integrate(x)Cos(x);}}`
+
+- In the console session, display an example:
+
+`In> {{Example();}}`
+
+- Quit from the console session:
+
+`In> {{quit}}`
+
+- Execute one or more yacas scripts (without terminal or prompts), then exit:
+
+`yacas -p -c {{path/to/script1}} {{path/to/script2}}`
```

Comment:
Any reason you didn't use the long options here? Especially in this case I think it would help remember them easier.

Reply:
They don't exist.

From the manual:

yacas -c
Inhibit printing of prompts In> and Out>. Useful for non-interactive sessions.

yacas -p
Does not use terminal capabilities, no fancy editing on the command line and no escape sequences printed. Useful for non-interactive sessions.

Although there are some long flags, there are no long versions of the short flags given in this reference.